app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt

```diff
@@ -237,6 +237,30 @@ class RecentlyPlayedFragmentTest {
     }
   }
 
+  @Test
+  fun testRecentlyPlayedTestActivity_fractionsPlayed_storiesForYouToolbarTitleIsDisplayed() {
```

Comment:
Do we have tests for the other toolbar cases? If not, please add them.

Reply:
Yes, it is present. I will rename other testcase to keep it consistent with this.

Reply:
Done.

Reply:
Ack.